db/migrate/20181012150410_create_book_suggestions.rb

```diff
@@ -0,0 +1,16 @@
+class CreateBookSuggestions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :book_suggestions do |t|
+      t.string :synopsis
```

Comment:
Maybe a `text` field should be a better option, shouldn't it?

Reply:
I was following what the card said but I agree.